test/framework/e2e_framework.go

```diff
@@ -598,6 +598,37 @@ func (f *E2eFramework) DumpClusterInfo(test string, namespace ...string) error {
 		if err := kubectl.DumpClusterInfo(opts); err != nil {
 			errs = append(errs, fmt.Errorf("failed to dump cluster info for cluster %s: %w", ctx, err))
 		}
+
+		// Store the list of pods in an easy to read format.
+		output, err := kubectl.Get(kubectl.Options{Context: ctx, Namespace: namespace[1]}, "pods", "-o", "wide")
+		if err != nil {
+			f.logger.Info("dump failed", "output", output, "error", err)
+			errs = append(errs, fmt.Errorf("failed to dump %s for cluster %s: %w", "pods", ctx, err))
+		}
+		f.storeOutput(outputDir, namespace[1], "pods", "out", output)
+
+		// Dump all objects that we need to investigate failures as a flat list and as yaml manifests
+		for _, objectType := range []string{"K8ssandraCluster", "CassandraDatacenter", "Stargate", "Reaper", "StatefulSet", "Secrets", "ReplicatedSecret", "ClientConfig"} {
```

Comment:
Can you add `CassandraTask` to this list?

Reply:
Sure thing, done 👍